.travis.yml

```diff
@@ -0,0 +1,36 @@
+dist: xenial
+language: python
+python: 3.7
+sudo: required
+env:
+  global:
+    - BUILD=true
+    - CHARM_BUILD_DIR=/tmp/charm-builds
+  matrix:
+    - JUJU_CHANNEL=stable
+    - JUJU_CHANNEL=edge
+matrix:
+  allow_failures:
+    - env: JUJU_CHANNEL=stable
+install:
+  - sudo apt purge libyaml-dev
+  - sudo snap install charm --classic
+  - sudo snap install juju --classic --channel $JUJU_CHANNEL
+  - sudo snap install juju-wait --classic
+  - sudo snap install microk8s --classic --channel 1.13/stable
```

Comment:
Any reason to pick `1.13` here and not the latest `stable`?

Reply:
Yeah, 1.14 has an issue that I think is related to this:

https://github.com/containerd/containerd/issues/2483

It looks like an upstream issue that will fix itself eventually. cc @ktsakalozos

Reply:
Got fixed by the excellent @ktsakalozos :cake: